tensorrt_llm/_torch/auto_deploy/transform/library/fuse_rope_mla.py

```
@@ -442,6 +441,14 @@ def _apply(
             ad_logger.info("No torch_mla nodes found; skipping.")
             return gm, TransformInfo(skipped=True, detail="no MLA nodes")
 
+        # Whether MLA RoPE weights are in native GPTJ layout (model skipped the
+        # de-interleave load hook) — if so, _undo_rope_deinterleave must NOT run.
+        try:
+            _model_config, _ = factory._get_model_config()
+            already_gptj = is_gptj_layout(_model_config)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)